src/site/_data/contributors.json

```diff
@@ -1510,5 +1510,13 @@
       "unit": "Developer Relations"
     },
     "homepage": "https://snugug.com"
+    },
+  "yoavweiss": {
+      "name": {
+        "given": "Yoav",
+        "family": "Weiss"
+      },
+      "twitter": "yoavweiss",
+      "github": "yoavweiss"
```

Comment:
Yoav uses this image on GitHub and Twitter so this one is probably good for his web.dev profile image: https://avatars0.githubusercontent.com/u/786187?s=400&v=4

Reply:
I tagged @robdodson on this because I can't figure out where we keep author images. Could be somewhere I don't have access to?

Reply:
oh sorry! those go in src/site/content/en/images/authors. Be sure to add a 2x version along with the regular version.

Reply:
@robdodson Done, thanks for the help 👍  

@kaycebasques I suggest we [update the handbook](https://web.dev/handbook/contributor-profile/#create-a-profile-image) with this folder path to avoid confusion because there's also `src/images` in this repo.